src/test/librbd/CMakeLists.txt

```diff
@@ -21,6 +20,11 @@ set(librbd_test
   test_Trash.cc
   journal/test_Entries.cc
   journal/test_Replay.cc)
+if(WITH_RBD_RWL)
+  list(APPEND librbd_test
+    test_WriteLogMap.cc
+    test_BlockGuard.cc)
```

Comment:
Isn't block guard used by other things? object map?

Reply:
It looks that way. At one point I think there were two BlockGuards, but not now.